PixivImage.py

```diff
@@ -375,7 +375,7 @@ def WriteInfo(self, filename):
         info.write("Tags          = " + ", ".join(self.imageTags) + "\r\n")
         info.write("Image Mode    = " + self.imageMode + "\r\n")
         info.write("Pages         = " + str(self.imageCount) + "\r\n")
-        info.write("Date          = " + self.worksDate + "\r\n")
+        info.write("Date          = " + self.root["createDate"] + "\r\n")
```

Comment:
same case in here, it supposed human readable.

Reply:
also will be converted to local time if it is set to config.ini

Reply:
I'd call "2020-10-12T07:50:09+00:00" easily human readable. For local time we could just use str(self.worksDateDateTime) which would be the same, sans "T" – although not much point in converting since the +00:00 is in there
But which format do you suggest otherwise?

Reply:
use `str(self.worksDateDateTime)` because if `useLocalTimezone` is enabled in the config, then at least it will be converted.